mender-deb-package

```diff
@@ -38,8 +38,18 @@ checkout_repo() {
 }
 
 get_deb_version() {
-  # Create a version from VCS. For master, generate something like 0.0~git20191022.dade697
-  DEB_VERSION=$(git describe --tags --exact-match 2>/dev/null || git log -1 --pretty=0.0~git%cd.%h --date format:%Y%m%d)-1
+  # Create a version from Git. For master, generate X.Y.Z~git20191022.dade697-1,
+  # where X.Y.Z is latest tag (not necessarily matching git describe)
+  if git describe --tags --exact-match 2>/dev/null; then
+    DEB_VERSION="$(git describe --tags --exact-match)-1"
+  else
+    DEB_VERSION="$(git tag | egrep '^[0-9]+\.[0-9]+\.[0-9]+$' | sort -rV | head -n1)"
+    if [ -z "${DEB_VERSION}" ]; then
+      DEB_VERSION="0.0"
```

Comment:
Not `0.0.0`?

Reply:
Before this PR we were using `0.0`, which is a notation that I saw in our non-versioned packages in Debian repos (mendertesting, log, scopstack), so I just followed the same.

However I run into a test error now (we have an strict regex to check the version) and it will be more convenient to use `0.0.0`, so I will change it :)